### PR TITLE
Updates for #MMS2019Docathon

### PR DESCRIPTION
**Pull request by AdamGrossTX**
Fixed 2 open issues #MMS2019Docathon

### CODE REVIEW

**Comment by PRMerger13** [comment]
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.It's easy to access the private repo version of your articles in GitHub. Just add -pr to the URL, as follows: 
 Public repo URL: https://github.com/Microsoft/azure-docs/blob/master/articles/azure-portal-overview.md 
 Private repo URL: https://github.com/Microsoft/azure-docs-pr/blob/master/articles/azure-portal-overview.md 
 We have closed this pull request. Please resubmit the updates through the private repo. If you need perms to the private repo, [see the contributor guide instructions](https://github.com/Microsoft/azure-docs/blob/master/contributor-guide/tools-and-setup.md#permissions)

**Comment by PRMerger13** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added or Deleted File(s) | This PR contains added or deleted file(s).
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added or Deleted File(s) | This PR contains added or deleted file(s).
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger15** [comment]
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.It's easy to access the private repo version of your articles in GitHub. Just add -pr to the URL, as follows: 
 Public repo URL: https://github.com/Microsoft/azure-docs/blob/master/articles/azure-portal-overview.md 
 Private repo URL: https://github.com/Microsoft/azure-docs-pr/blob/master/articles/azure-portal-overview.md 
 We have closed this pull request. Please resubmit the updates through the private repo. If you need perms to the private repo, [see the contributor guide instructions](https://github.com/Microsoft/azure-docs/blob/master/contributor-guide/tools-and-setup.md#permissions)

**Comment by ChrisKibble** [comment]
@AdamGrossTX / @aczechowski, suggested edits before you go live for **sccm-ps/ConfigurationManager/New-CMTSRule.md**:

- [ ] Line 27 - Add a space after ## so that the markdown creates the heading you're looking for.
- [ ] Line 67 - Change C:\> to XYZ:\> I was working on making this update across most of the pages to show that ConfigMgr cmdlets can't be run from C:.

**Comment by aczechowski** [comment]
This fixes #48
#MMS2019Docathon
I'm going to try reopening this one more time. If that still doesn't work, we'll come back to it next week to figure out a game plan.

**Comment by PRMerger6** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added or Deleted File(s) | This PR contains added or deleted file(s).
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger6** [comment]
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.It's easy to access the private repo version of your articles in GitHub. Just add -pr to the URL, as follows: 
 Public repo URL: https://github.com/Microsoft/azure-docs/blob/master/articles/azure-portal-overview.md 
 Private repo URL: https://github.com/Microsoft/azure-docs-pr/blob/master/articles/azure-portal-overview.md 
 We have closed this pull request. Please resubmit the updates through the private repo. If you need perms to the private repo, [see the contributor guide instructions](https://github.com/Microsoft/azure-docs/blob/master/contributor-guide/tools-and-setup.md#permissions)

File: sccm-ps/ConfigurationManager/Set-CMDistributionPointGroup.md [hunk 1]
```diff
@@ -35,20 +35,11 @@ The **Set-CMDistributionPointGroup** cmdlet changes the configuration settings o
 
 ## EXAMPLES
 
-### Example 1: Add a distribution point group to a security scope
+### Example 1: Rename Distribution Point Group and add a Description
```
Comment (aczechowski) [inline sccm-ps/ConfigurationManager/Set-CMDistributionPointGroup.md]: Use sentence case:

```suggestion
### Example 1: Rename a distribution point group and add a description
```

File: sccm-ps/ConfigurationManager/New-CMTSRule.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
+---
```
Comment (aczechowski) [inline sccm-ps/ConfigurationManager/New-CMTSRule.md]: Metadata block should include the following lines starting on line 1:

```
---
title: New-CMTSRule
titleSuffix: Configuration Manager
description: Creates a new rule on a task sequence step in Configuration Manager.
ms.date: 05/07/2019
ms.prod: configuration-manager
ms.technology: configmgr-other
ms.topic: conceptual
author: aczechowski
ms.author: aaroncz
manager: dougeby
---
```

I'm hoping that by setting me as author, our automation bot will let us reopen this PR! 🤞

File: sccm-ps/ConfigurationManager/New-CMTSRule.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+ [-DisableWildcardHandling] [-ForceWildcardHandling] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## Description
+
+## EXAMPLES
+
+### Example 1: Sets a task sequence variable to a value
+```
+PS C:\> New-CMTSRule -Variable @{"MyTSVar" = 12345678}
```
Comment (aczechowski) [inline sccm-ps/ConfigurationManager/New-CMTSRule.md]: as Chris Kibble suggested, change the drive to `PS XYZ:\>`

File: sccm-ps/ConfigurationManager/New-CMTSRule.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+# manager: MSFT-alias-of-manager-or-PM-counterpart
+external help file: AdminUI.PS.AppMan.dll-Help.xml
+---
+# New-CMTSRule
+
+## SYNOPSIS
+
+Creates a new rule that can be added to task sequence steps.
+
+##SYNTAX
```
Comment (aczechowski) [inline sccm-ps/ConfigurationManager/New-CMTSRule.md]: Add a space to fix header:

```suggestion
## SYNTAX
```

File: sccm-ps/ConfigurationManager/New-CMTSRule.md [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
+## INPUTS
```
Comment (aczechowski) [inline sccm-ps/ConfigurationManager/New-CMTSRule.md]: If these remaining sections are blank, let's just remove them. Or can comment them out: `<!-- stuff in here -->`

**Comment by aczechowski** [comment]
@AdamGrossTX, a similar process worked with a new article by Chris Kibble, so I'm optimistic that the metadata changes at least I mentioned will allow this pull request to open again 😄

**Comment by AdamGrossTX** [comment]
@aczechowski I have been looking for this PR for a while. I remembered reading through these and then lost it. Anyway the PR is still closed and I can't make any edits to this one based on your suggestions. What should we do here?

**Pull request closed without merging**